src/goto-instrument/document_properties.cpp

```diff
@@ -155,9 +155,9 @@ void document_propertiest::get_code(
   dest="";
 
   const irep_idt &file=source_location.get_file();
-  const irep_idt &line=source_location.get_line();
+  const irep_idt &source_line=source_location.get_line();
```

Comment:
You could clang-format this line at the same time.

Reply:
Will do!